coop_membership/models/shift_shift.py

```diff
@@ -1,33 +1,33 @@
-# -*- coding: utf-8 -*-
 # Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
 # @author: La Louve
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html
 
-from openerp import api, fields, models, _
-from openerp.exceptions import UserError
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
 from datetime import datetime, timedelta
 
 
 class ShiftShift(models.Model):
     _inherit = "shift.shift"
 
-    standard_registration_ids = fields.One2many(
-        "shift.registration",
-        "shift_id",
-        string="Standard Attendances",
-        domain=[('shift_type', '=', 'standard')])
+    standard_registration_ids = fields.One2many("shift.registration",
+                                                "shift_id",
+                                                string="Standard Attendances",
+                                                domain=[('shift_type', '=',
+                                                         'standard')])
```

Comment:
readability was much better before :sweat_smile: 
I don't understand this change

It was already OCA-style compliant, and now it's not